lib/rage/cli.rb

```diff
@@ -112,6 +113,29 @@ def console
       IRB.start
     end
 
+    desc "middleware", "List Rack middleware stack enabled for the application"
+    def middleware
+      app = ::Rack::Builder.parse_file(options[:config] || "config.ru")
+      app = app[0] if app.is_a?(Array)
+      middlewares = []
+
+      while app
+        if app.instance_variable_defined?(:@app)
+          middlewares << app.class
+          app = app.instance_variable_get(:@app)
+        else
+          break
+        end
+      end
+
+      middlewares.each { |middleware| puts middleware }
```

Comment:
Can we make it look a bit closer to the output of `rails middleware`?

Reply:
@rsamoilov - Thank you for the comment. I think the line will produce a similar output. What do you think?
```
say "use #{middleware.first.inspect}"
```

Reply:
Looks good. Probably calling `name` would be a bit more clear.

```
say "use #{middleware.first.name}"
```